platform/android/api/api.h

```diff
@@ -28,5 +28,10 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#ifndef API_H
```

Comment:
I would name it ANDROID_API_H to ensure uniqueness. There's also a similar one for JavaScript that might be worth checking.

Reply:
I checked and `jplatform/javascript/api/api.h` has the same problem indeed.